test/symlink.js

```diff
@@ -84,7 +97,7 @@ describe("symlink", function() {
 			[path.join(tempPath, "that"), "./test/temp/lib/node.js", "with symlinked directory as context and in path (chained)"],
 			[path.join(tempPath, "that", "lib"), "./node.js", "with symlinked directory in context path (chained)"],
 			[path.join(tempPath, "that", "test"), "./temp/node.js", "with symlinked directory in context path and symlinked file (chained)"],
-			[path.join(tempPath, "that", "test"), "./temp/lib/node.js", "with symlinked directory in context path and symlinked directory (chained)"],
+			[path.join(tempPath, "that", "test"), "./temp/lib/node.js", "with symlinked directory in context path and symlinked directory (chained)"]
```

Comment:
was the trailing comma removed due to a linting error? if not, please restore it to maintain the original code style